project_euler/validate_solutions.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+import importlib.util
+import logging
+import pathlib
+
+LOG_FILENAME = "project_euler_test.log"
+
+# Keep the answer part in str for consistency as some problems could output in str
+# Problem 329: 199740353/29386561536000 (Format for this answer is a/b)
+# https://github.com/nayuki/Project-Euler-solutions/blob/master/Answers.txt#L191
+ANSWERS = {
+    1: "233168",
+    2: "4613732",
+    3: "6857",
+    4: "906609",
+    5: "232792560",
+    6: "25164150",
+    7: "104743",
+    8: "23514624000",
+    9: "31875000",
+    10: "142913828922",
+    11: "70600674",
+    12: "76576500",
+    13: "5537376230",
+    14: "837799",
+    15: "137846528820",
+    16: "1366",
+    17: "21124",
+    18: "1074",
+    19: "171",
+}
+
+WORKING_DIR = pathlib.Path.cwd()
+
+if WORKING_DIR.name == "Python":
+    PROJECT_EULER_PATH = WORKING_DIR.joinpath("project_euler")
+# Below two checks are just in case something goes wrong
+# Travis root dir = /home/travis/build/TheAlgorithms/Python
+# https://travis-ci.com/github/TheAlgorithms/Python/builds/186187397#L398
+elif WORKING_DIR.name == "TheAlgorithms":
+    PROJECT_EULER_PATH = WORKING_DIR.joinpath("Python", "project_euler")
+elif WORKING_DIR.name == "project_euler":
+    PROJECT_EULER_PATH = WORKING_DIR
```

Comment:
I believe even this won't be required, it was there only for precaution. It seems Travis is picking up the path consistently as the root dir mentioned in the comments above.